src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Index.cshtml

```diff
@@ -31,52 +33,140 @@
             </tr>
             </thead>
             <tbody>
-            <tr>
-                <td>
-                    <input type="text" class="form-control" asp-for="CustomRedirect.OldUrl">
-                    <span asp-validation-for="CustomRedirect.OldUrl" class="text-danger"></span>
-                </td>
-                <td>
-                    <input type="text" class="form-control" asp-for="CustomRedirect.NewUrl">
-                    <span asp-validation-for="CustomRedirect.NewUrl" class="text-danger"></span>
-                </td>
-                <td class="text-center align-middle">
-                    <input type="checkbox" class="form-check-input" asp-for="CustomRedirect.WildCardSkipAppend">
-                </td>
-                <td>
-                    <select class="form-select" asp-for="CustomRedirect.RedirectType">
-                        <option selected value="@RedirectType.Temporary">@RedirectType.Temporary</option>
-                        <option value="@RedirectType.Permanent">@RedirectType.Permanent</option>
-                    </select>
-                </td>
-                <td>
-                    <div class="d-grid gap-2">
-                        <button type="submit" class="btn btn-primary"
-                                asp-page-handler="create">
-                            <span data-feather="plus"></span> add
-                        </button>
-                    </div>
-                </td>
-            </tr>
-            @foreach (var item in Model.Items)
+            @if (!Model.IsEditing())
             {
-                <tr class="align-middle">
-                    <td>@item.OldUrl</td>
-                    <td>@item.NewUrl</td>
+                <tr>
+                    <td>
+                        <input type="text" class="form-control" asp-for="CustomRedirect.OldUrl">
+                        <span asp-validation-for="CustomRedirect.OldUrl" class="text-danger"></span>
+                    </td>
+                    <td>
+                        <input type="text" class="form-control" asp-for="CustomRedirect.NewUrl">
+                        <span asp-validation-for="CustomRedirect.NewUrl" class="text-danger"></span>
+                    </td>
                     <td class="text-center align-middle">
-                        @await Component.InvokeAsync("CheckboxReadonly", new { isChecked = item.WildCardSkipAppend })
+                        <input type="checkbox" class="form-check-input" asp-for="CustomRedirect.WildCardSkipAppend">
+                    </td>
+                    <td>
+                        <select class="form-select" asp-for="CustomRedirect.RedirectType">
+                            <option selected value="@RedirectType.Temporary">@RedirectType.Temporary</option>
```

Comment:
There was another PR that allowed to set default redirect type. I think this should be adjusted too.